docs/ref/templates/builtins.txt

```diff
@@ -2068,7 +2068,7 @@ For example:
 
 .. code-block:: html+django
 
-    {{ value|iriencode }}
+    {{ value|urlencode|iriencode }}
 
 If ``value`` is ``"?test=1&me=2"``, the output will be ``"?test=1&amp;me=2"``.
```

Comment:
The current example is not incorrect, as auto-escaping behavior will quote ``"?test=1&me=2"`` to ``"?test=1&amp;me=2"``, but it has nothing to do with `iriencode`.

Reply:
I see. Thank you for the clarification, making the change at the moment!